features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha_alt_hw.h

```diff
@@ -51,58 +43,51 @@ void crypto_sha_update(crypto_sha_context *ctx, const unsigned char *input, size
 void crypto_sha_update_nobuf(crypto_sha_context *ctx, const unsigned char *input, size_t ilen, int islast);
 void crypto_sha_getinternstate(unsigned char output[], size_t olen);
 
-#if defined(MBEDTLS_SHA1_C)
 #if defined(MBEDTLS_SHA1_ALT)
 
 void mbedtls_sha1_hw_init( crypto_sha_context *ctx );
 void mbedtls_sha1_hw_free( crypto_sha_context *ctx );
 void mbedtls_sha1_hw_clone( crypto_sha_context *dst,
-                         const crypto_sha_context *src );
```

Comment:
I don't think it's sensible to have a H/W-to-H/W cloning function, and this function is indeed not used in the current code (in particular, it's not called from the wrapper function `mbedtls_sha1_clone`). @ccli8 Do you know why this was introduced?
As this was introduced in an earlier PR and is public, though, I suspect we must not change it now.

Reply:
I remove these H/W clone functions in https://github.com/ARMmbed/mbed-os/pull/4925/commits/9f35e2f90fe5dc4830a4bf7dc1a3c4d13e2ff860.